.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
nit: please add the standard OpenThread license header

Reply:
Where can I find the latest version? I different files contain slightly different versions (years, names, etc.)

Also, the whole OpenThread project is under BSD 3-Clause license as declared in the [LICENSE](https://github.com/openthread/openthread/blob/main/LICENSE) file right? It's quite uncommon to have the full license as a header for each file, can I read up on the motivation for that somewhere?

Reply:
You can find an example in https://github.com/openthread/openthread/blob/main/.github/workflows/build.yml, just update the year to 2022.

BSD 3-Clause is short enough that it is feasible to be included at the top of every file. FWIW, FreeBSD itself follows the same model.

Reply:
Thanks, added!

Feel free to squash when merging.